pquery-run.sh

```diff
@@ -612,7 +625,7 @@ pxc_startup() {
     echo "ssl-cert = ${WORKDIR}/cert/server-cert.pem" >> ${DATADIR}/n${i}.cnf
     echo "ssl-key = ${WORKDIR}/cert/server-key.pem" >> ${DATADIR}/n${i}.cnf
     if [ "$IS_STARTUP" == "startup" ]; then
-      ${MID} --datadir=$node > ${WORKDIR}/startup_node1.err 2>&1 || exit 1
+      ${INIT_TOOL} ${INIT_OPT} --basedir=${BASEDIR} --datadir=$node > ${WORKDIR}/startup_node1.err 2>&1
```

Comment:
Is this in similar in code to the revised MD startup elsewhere in the script?

Reply:
i.e. does it match in code with standard MD startup, or is a different setup (considering uniformity)